app/src/main/res/layout/fragment_ciam_mode.xml

```diff
@@ -0,0 +1,158 @@
+
+                <TextView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="3"
+                    android:layout_gravity="center_vertical"
+                    android:textStyle="bold"
+                    android:text="Scope" />
+
+                <LinearLayout
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical"
+                    android:layout_weight="7">
+
+                    <EditText
+                        android:id="@+id/scope"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:text="https://substrate.office.com/profile//User.Read"
+                        android:textSize="12sp" />
+
+                    <TextView
+                        android:layout_height="wrap_content"
+                        android:layout_width="match_parent"
+                        android:paddingLeft="5dp"
+                        android:text="Type in scopes delimited by space"
+                        android:textSize="10sp"  />
+
+                </LinearLayout>
+            </LinearLayout>
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                android:paddingTop="5dp"
+                android:paddingBottom="5dp"
+                android:weightSum="10">
+
+                <TextView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="3"
+                    android:text="Accounts"
+                    android:textStyle="bold" />
+
+                <Spinner
+                    android:id="@+id/account_list"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="7"
+                    android:spinnerMode="dropdown" />
+            </LinearLayout>
+
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                android:paddingTop="5dp"
+                android:paddingBottom="5dp"
+                android:weightSum="10">
+
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center_vertical"
+                    android:layout_weight="3"
+                    android:text="At this moment, CIAM does not support MS Graph, so we cannot call the graph API in this fragment. Instead we will simply output the access token after a succesful request."
```

Comment:
not-that-important-nit, since I'm noticing the strings in other layout files too, but if you do end up editing this file again, I think it would be good to create a string resources file, esp when we have long strings like this one.

Reply:
Good call!